### PR TITLE
Minor annual report

### PR DESCRIPTION
**Pull request by JRPearson500**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -14,19 +14,19 @@ The NHS England Digitial Analytics and Research Team *(formerly NHSX Analytics U
 Within the unit we have analysts, data engineers, data scientists, developers, economists, and evaluation specialists.   Here we will advertise public projects from all these disciplines as well as publishing thoughts on how healthcare analysts can interact with the open tools and data.
```
Comment (danjscho) [inline index.md:14]: unit -> team?

File: index.md [hunk 2]
```diff
@@ -14,19 +14,19 @@ The NHS England Digitial Analytics and Research Team *(formerly NHSX Analytics U
...
+The second pillar is centred around our belief that unstructured data (especially text data) has huge unlocked potential for healthcare analysis.  A key blocker to using unstructured text data is the potential sensitivity within the data.  Often this is partially dealt with by anonymising for patient identifiable information but this can still leave content which is contextual or domain specific and therefore still disclosive.  We are therefore aiming to **highlight the privacy of unstructured data beyond PII** through understanding the problem and currently available options and then creating tool to calculate a privacy fingerprint for a data base or piece of text.  This fingerprint would be a structured way of addressing the risk of disclosure which could then be used in an IG process to understand what masking/anonymisation is required or what limited terms of use need to be applied to the data.  By having a tangible way of addressing this issue we aim to increase the sharing of these data with confidence.
```
Comment (danjscho) [inline index.md:24]: Possible link to the `Functional Anonymisation Paper` (e.g. https://eprints.soton.ac.uk/417832/1/Functional_Anonymisation_and_the_Data_Environment_Final.pdf) which gives a nice overview of challenges with anonymisation in general?

File: index.md [hunk 3]
```diff
@@ -14,19 +14,19 @@ The NHS England Digitial Analytics and Research Team *(formerly NHSX Analytics U
...
+
+our fourth research area is a common area for data science but still with large unmet need for healthcare - **increasing the available validation methods for model application**.  We have previously seen the value of having defined benchmarks and gold standard downstream tasks for evaluation and validation and we aim to investigate what currently available methods allow us to identify and discuss fairness across our different models.
```
Comment (danjscho) [inline index.md:28]: our -> Our

**Review by danjscho — CHANGES_REQUESTED** [review]
Some minor typos and possible link to add.

File: index.md (hunk 1 above)
Comment (JRPearson500) [reply index.md]: done

File: index.md (hunk 3 above)
Comment (JRPearson500) [reply index.md]: done

**Pull request merged**